src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+export class TechLoad {
+
+  public jobs: JobData[];
+  public jobCount: number;
+  public uuid: string;
+
+  private seed: number;
+  private seedIndex: number = 0; // doesn't really matter what it's instantiated to, just needs to be any number >= 0 so the seed methods can generate the next number
+
+  constructor(uuid: string, amount?: number) { // Can be built with a given amount, but if not will seed the number of jobs to build
+    this.uuid = uuid;
+    this.seed = Math.abs(this.convertStrToNum(uuid));
+    
+    this.jobs = [];
+    // generate a random number of jobs, weighted towards ~3
+    const generatedJobCount = Math.round( (this.nextRand()*7 + this.nextRand()*7) / 2 );  // This line needs to run regardless of if we were passed an amount, or else the seeded rng will get messed up!
+    this.jobCount = amount ? amount : generatedJobCount; // allows for front end to request a specific number of jobs. If amount is not present, use the above generated number
+    for (let i = 0; i < this.jobCount; i++) {
+      this.jobs.push(this.generateJob()); // Where all the "magic" happens, generateJob will create all job data and push it into job array
+    }
+  }
+
+  /**
+   * @description - Called by constructor to generate a job. 
+   * @returns {JobData} - Returns an object which contains the entirety of job data required to fill out a fake ticket
+   */
+  private generateJob(): JobData {
+    // The following variables are generated prior to the output object, as they are passed into generator methods in order to generate data consistent throughout the ticket
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
+    const streetAddress = this.generateStreetAddress();
+    const jobType = this.randOf(['Install', 'Repair', 'Helper', 'POTS', 'BSW']);
+    const transportType = jobType === 'POTS' || this.nextRand() < 0.5 ? this.randOf(['IP-CO', 'IP-RT', 'FTTN', 'FTTN-BP']) : this.randOf(['FTTP GPON', 'FTTP XGS-PON']);
+    const curJob: JobData = {
+      firstName: firstName,
```

Comment:
You could just put `firstName,`. Such with last name

Reply:
This seems like something Atlas wouldn't like but I changed it.